lib/benchpark/openmp.py

```diff
@@ -9,16 +9,11 @@
 
 
 class OpenMPExperiment:
-    variant(
-        "openmp",
-        default="non",
-        values=("oui", "non"),
-        description="Build and run with OpenMP",
-    )
+    variant("openmp", default=False, description="Build and run with OpenMP")
 
     class Helper(ExperimentHelper):
         def get_helper_name_prefix(self):
-            return "openmp" if self.spec.satisfies("openmp=oui") else ""
+            return "openmp" if self.spec.satisfies("openmp") else ""
```

Comment:
This would have to be `.satisfies("+openmp")` wouldn't it?

(if this is true, then FYI this correction needs to be made in several files)